data/scripts/creaturescripts/modal_window_helper.lua

```diff
@@ -0,0 +1,197 @@
+local ModalWindowAutoID = 10000
+local buttonOrder = {
+    [4] = {2, 3, 4, 1},
+    [3] = {2, 3, 1},
+    [2] = {2, 1},
+    [1] = {1}
```

Comment:
What are those numbers?

Reply:
It is an arbitrary way of ordering the buttons according to the number of buttons, at most only 4 buttons can be displayed if I am not mistaken, however I agree with eliminating this part and that it be done by default